### PR TITLE
Tobe reviewed

### PR DESCRIPTION
**Pull request by NatanimA**
-->Inserted Linters
-->Coppied the project inside this new repo
--->Edited Readme

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#main {
+    margin: 30px 8px;
+    padding: 15px;
+    border-radius: 5px;
+    background: #eee;
```
Comment (NatanimA) [inline style.css:5]: please use linters for correction you can run  "npx stylelint "**/*.{css,scss}" to see what changes you can make. 😉

File: style.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+#img-div {
+    background: white;
+    padding: 10px;
+    margin: 0;
+}
+figcaption {
+    text-align: center;
+    font-size: 0.9rem;
+    /* width: 95%; */
+}
```
Comment (NatanimA) [inline style.css:27]: Same here you can use two indentations as the rule stated in the linters. and please remove unused codes on line 26. 😃

File: style.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        text-align: -webkit-match-parent;
+    }
+    strong {
+        font-weight: bold;
+    }a:visited {
+        color: #74638f;
+    }
+    blockquote {
+        font-style: italic;
+        max-width: 545px;
```
Comment (NatanimA) [inline style.css:54]: Here as well you can change it by following the linters.😁

File: style.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    #tribute-link{ 
+         margin: 0px 50px;
+         text-align: center;
+         font-size: 1.4rem;
+         font-weight: bold;
+                           
+}
+
+    
+
```
Comment (NatanimA) [inline style.css:67]: you only need one extra line after you finished a block ,this is a liter error as well , please try to follow the linter rules as much as you can. 😄

File: index.html [hunk 5]
```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Tribute Page</title>
+  <link rel="stylesheet" href="style.css">
```
Comment (NatanimA) [inline index.html:6]: Please include the meta view port as stated in the linter errors you can check errors occurred by running "npx hint ."

File: index.html [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+          </li>
+          <li>
+              <strong>2009</strong>
+              - dies at the age of 95.
+          </li>
+        </ul>
+
+        <blockquote cite="https://borlaug.cfans.umn.edu/unparalleled-achievements#:~:text=For%20his%20innumerable%20and%20unparalleled,(2005)%2C%20and%20the%20Congressional">
+          "Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress."
+          <br>
```
Comment (NatanimA) [inline index.html:94]: Please insert the correct Url for the caption 😄

**Review by NatanimA — COMMENTED** [review]
Required Changes ♻️

[](https://camo.githubusercontent.com/0b6134a224332efd68cb15c3a3ff40ac036be75801f5ab22f0905d6b343c8a60/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f714c57644d5958314e594632672f67697068792d646f776e73697a65642d6c617267652e676966)

**Review by sumairq — APPROVED** [review]
### Hi @NatanimA    , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Great Communication
✔️ Friendly tone
✔️Made good inline comments🥇
✔️Awesome review summary

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**